update-source-git/README.md

```diff
@@ -0,0 +1,105 @@
+`packit source-git update-source-git`
+
+This research covers a command which synchronizes changes from dist-git back to
+a source-git repo.
+
+Areas to cover:
+
+- spec file synchronization
+
+- sources synchronization
+
+- `.distro/source-git.yaml` update
+
+- code synchronization
+
+Let's break these down.
+
+## Spec file sync
+
+This seems trivial, we can copy the file from dist-git back to source-git. But
+before we do that, we should capture the version field and check if it differs
+from the source-git spec.
```

Comment:
Should the release field also be considered?

What action should be taken if the version field is different?

Reply:
what do you mean by considered?

I'll add a note about the version change.

Reply:
I mean: should there be some action taken if the release fields in dist-git and source-git are different?

Reply:
okay, thanks for the explanation, I'll add a note about it